model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -393,8 +393,8 @@ message StandardPTransforms {
     // https://s.apache.org/beam-runner-api-combine-model#heading=h.h5697l1scd9x
     // Payload: CombinePayload
     COMBINE_PER_KEY_CONVERT_TO_ACCUMULATORS = 4
-        [(beam_urn) =
-             "beam:transform:combine_per_key_convert_to_accumulators:v1"];
+    [(beam_urn) =
```

Comment:
I should revert whitespace changes here. I'll do that when current triggered tests finished.

Reply:
Done.